registry.json

```diff
@@ -1,4 +1,20 @@
 [
+  {
+      "ticker": "SWIFT",                                                      
+      "poolId": "pool1mqez65p7fvq29qrf8skwxjw0zlq0mar62c6v0m5kn95tgzeh7rv",
```

Comment:
Please can you change this to pool id? I think yours is d8322d503e4b00a280693c2ce349cf17c0fdf47a5634c7ee969968b4 ?

Reply:
Hi. Have changed the pool id.